documentation/manual/src/ornate/tutorial.md

```diff
@@ -580,7 +580,7 @@ and since we are only describing the endpoint, we added an abstract implicit met
 The response of the `getMeter` endpoint is described by the expression
 `option(jsonResponse[Meter])`, which means that it
 can optionally be empty (for instance if a client queries this endpoint with a
-non-existing `UUID`). The `option` method, provided by the `OptionalResponses` algebra
+non-existing `UUID`). The `option` method, provided by the `Responses` algebra
```

Comment:
I would just drop the “, provided by the `Responses` algebra interface, ” because now it is part of the basic `Endpoints` algebra interface.

Reply:
done